main.py

```diff
@@ -1,10 +1,69 @@
+import json
+from typing import Optional
+
 import init_django_orm  # noqa: F401
+from django.db import IntegrityError
 
 from db.models import Race, Skill, Player, Guild
 
 
+def load_data(filepath: str) -> dict:
+    with open(filepath) as file:
+        return json.load(file)
+
+
+def create_or_get_race(race_data: dict) -> Race:
+    race, _ = Race.objects.get_or_create(
```

Comment:
this function basically does one thing so it doesn't make much sense to create a funciton for it

Reply:
You can basically do this instead 
`race, _ = Race.objects.get_or_create(name=race_data["name"], description=race_data["description"])`